source/common/protobuf/utility.cc

```diff
@@ -48,9 +48,16 @@ ProtoValidationException::ProtoValidationException(const std::string& validation
   ENVOY_LOG_MISC(debug, "Proto validation error; throwing {}", what());
 }
 
+bool MessageUtil::allow_unknown_fields = false;
+
 void MessageUtil::loadFromJson(const std::string& json, Protobuf::Message& message) {
+  MessageUtil::loadFromJsonCustom(json, message, false);
+}
+
+void MessageUtil::loadFromJsonCustom(const std::string& json, Protobuf::Message& message,
```

Comment:
`loadFromJsonEx` ? (Not sure what custom means exactly)

Reply:
Done